spec/models/public_body_spec.rb

```diff
@@ -140,6 +140,19 @@
 
   end
 
+  describe '#expire_requests' do
+
+    it 'calls expire on all associated requests' do
+      authority = FactoryGirl.create(:public_body)
+      request = FactoryGirl.create(:info_request, :public_body => authority)
+      expect(authority).to receive(:info_requests).and_return([request])
```

Comment:
I don't think it's necessary to mock this, you might just need to reload the public body? (You could just create a request and then get the body out afterwards?)